packages/core/src/events/interfaces.ts

```diff
@@ -38,71 +38,136 @@ export interface CoreOptions {
   [key: string]: any
 }
 
-interface CoreAnalyticsContext {
+/**
+ * Context is a dictionary of extra information that provides useful context about a datapoint, for example the user’s ip address or locale. You should only use Context fields for their intended meaning.
+ * @link https://segment.com/docs/connections/spec/common/#context
+ */
+export interface CoreAnalyticsContext {
+  /**
+   * This is usually used to flag an .identify() call to just update the trait, rather than "last seen".
+   */
+  active?: boolean
+
+  /**
+   * Current user’s IP address.
+   * (node only, collected automatically in browser).
+   */
+  ip?: string
+
+  /**
+   * 	Locale string for the current user, for example en-US.
+   *  (node only, collected automatically in browser).
+   *  @example en-US
+   */
+  locale?: string
+  /**
+   * Dictionary of information about the user’s current location.
+   * Generally useful for mobile.
+   */
+  location?: {
+    /**
+     * @example San Francisco
+     */
+    city?: string
+    /**
+     * @example United States
+     */
+    country?: string
+    /**
+     * @example 40.2964197
+     */
+    latitude?: string
+    /**
+     * @example -76.9411617
+     */
+    longitude?: string
+    region?: string
```

Comment:
This is listed in the docs, but there is no example of this.

Reply:
Just my guess, but I'd see `region` as a more global friendly version of `state`. Province, State, whatever else you call the equivalent.